returnn/tf/layers/basic.py

```diff
@@ -1176,6 +1176,114 @@ def transform_config_dict(cls, d, network, get_layer):
       d["position"] = get_layer(d["position"])
 
 
+class SliceNdLayer2(_ConcatInputLayer):
```

Comment:
Why `2`? Never do that. Extend/generalize the existing `SliceNdLayer`.

Reply:
Will change the name and clean the rest once it is working. Let them both for comparison reasons.

Reply:
But you should not reimplement the whole function/layer but just extend it. I would assume this to be only a small or even almost minimal change to make it work.